resources/api-docs/1.0/default-images.md

```diff
@@ -0,0 +1,75 @@
+# Images
+
+---
+
+- [All Default Images](#all-images)
+- [Create a Image](#create-image)
+- [Delete a Image](#delete-image)
+
+<a name="all-images"></a>
+## All Images
+
+You can get a list of all the default images of a campaign by using the following endpoint. This is a superboosted campaign feature! If the campaign isn't superboosted, this API endpoint will result in a 404.
```

Comment:
Use the premium terminology as no one new to kanka since summer 2023 will have any idea what superboosted means.